lib/charms/archive_auth_mirror/reprepro.py

```diff
@@ -0,0 +1,40 @@
+import getpass
+
+from charmhelpers.core.templating import render
+
+from .utils import get_paths
+
+
+def configure_reprepro(mirror_uri, mirror_archs, mirror_key_fingerprint,
+                       sign_key_fingerprint, get_paths=get_paths):
+    '''Create reprepro configuration files.'''
```

Comment:
Huh? Triple single quotes are a no-no

as per https://www.python.org/dev/peps/pep-0008/#string-quotes and https://www.python.org/dev/peps/pep-0257/